tests/storage/storage.go

```diff
@@ -1031,9 +1036,14 @@ var _ = SIGDescribe("Storage", func() {
 			// Not a candidate for NFS because local volumes are used in test
 			It("[test_id:1015]should be successfully started", func() {
 				// Start the VirtualMachineInstance with the PVC attached
-				vmi = tests.NewRandomVMIWithPVC(dataVolume.Name)
+				vmi = libvmi.New(
+					libvmi.RandName(libvmi.DefaultVmiName),
+					libvmi.NewCirrosOptions(
+						libvmi.WithPersistentVolumeClaim(dataVolume.Name),
+						libvmi.WithInterface(libvmi.InterfaceDeviceWithMasqueradeBinding()),
+						libvmi.WithNetwork(virtv1.DefaultPodNetwork()))...)
+				//tests.NewRandomVMIWithPVC(dataVolume.Name)
```

Comment:
Is this correct?

Reply:
I don't know, unfortunately it does not work yet. I plan on returning to work on it tomorrow.

I wanted to replace `NewRandomVMIWithPVC` and found a VM needs interface and network, if you see anything else is missing feel free to tell me.

Reply:
I don't think you need to specify the interface. It's needed only if you want to not do functional change here. 
Do you fail 100%? It seems to me you were just unlucky.

I actually have similar work locally. I will try to push it.

Reply:
Yes, 100% failure when trying on local build. 
Cannot log in.

Reply:
Ok, found the problem, it was the order of disks, so far I have rebased and push changes for testing. So I can see if everything else works, and I'll try to clean up the PR.

Reply:
Can you elaborate?

Reply:
Disk with cloudInit has to be last, after the disk with PVC/DV. I am not sure why. 

https://github.com/kubevirt/kubevirt/blob/892314aa31091629c127c81aa9e64001bffe8386/tests/libvmi/factory.go#L77

Reply:
This is fixed. The order of disks/boot order